src/main/java/appeng/tile/AEBaseTile.java

```diff
@@ -488,7 +490,21 @@ public void disableDrops()
 
 	public void saveChanges()
 	{
-		markDirty();
+		if( this.world != null )
+		{
+			if( !this.markDirtyQueued )
+			{
+				TickHandler.INSTANCE.addCallable( null, this::markDirtyAtEndOfTick );
+				this.markDirtyQueued = true;
+			}
+		}
+	}
+
+	private Object markDirtyAtEndOfTick( final World w )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "w". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1172)